composer.json

```diff
@@ -11,7 +11,8 @@
     ],
     "require": {
         "php": ">=7.4 <8.3",
-        "latte/latte": "^2.11.3"
+        "latte/latte": "^2.11.3",
```

Comment:
co z latte sa pouziva v tomto balicku? vieme ho dat bud prec alebo navysit na ^3.0?

Reply:
to bolo mozno kvoli readme?? ale skor tam chybalo `nette/http`